DOCS/man/options.rst

```diff
@@ -5211,6 +5211,9 @@ The following video options are currently all specific to ``--vo=gpu`` and
     flipping GL front and backbuffers immediately (i.e. it doesn't call it
     in display-sync mode).
 
+    On OSX this is always deactivated because it only causes performance
```

Comment:
Is it really necessary to completely nerf this? Can't it just be off by default?

Reply:
that would either mean to change the default for all platforms or just macOS. former shouldn't be done and latter would mean more platform specific code in the common code, which should be avoided as far as i understand.

i am not quite sure how to handle it then.

Reply:
ok, fair enough